R/RT_functions.R

```diff
@@ -208,20 +203,6 @@ send_correspondence_at_time_x <- function(awards_db,
   
 }
 
-  
-## helper function to check RT replies
```

Comment:
Thanks Chris! Will definitely do that going forward. My apologies for having you review this without much context.

This is an issue with the `RT` package. I checked the most recent version here for the [`rt_ticket_history_reply` ](https://github.com/NCEAS/rt/blob/master/R/rt_ticket_history_reply.R) function and it now only returns the ticket id and not all the information as before (ie. `reply$status_code`). Therefore, the `check_rt_reply` no longer works as intended because it is dependent on the other information that is returned in older versions of this function.

Reply:
tried to fixed in subsequent updates